yt_dlp/extractor/youtube/_video.py

```diff
@@ -253,6 +258,8 @@ class YoutubeIE(YoutubeBaseInfoExtractor):
     _SUBTITLE_FORMATS = ('json3', 'srv1', 'srv2', 'srv3', 'ttml', 'srt', 'vtt')
     _DEFAULT_CLIENTS = ('tv', 'ios', 'web')
```

Comment:
Should these be renamed to _DEFAULT_UNAUTHED_CLIENTS, _DEFAULT_AUTHED_FREE_CLIENTS and _DEFAULT_AUTHED_PREMIUM_CLIENTS?

I'm mindful that we might soon need to split it further (eg separate "made for kids" and non-kids clients), so I think it would be good to keep the names as descriptive as possible